package.json

```diff
@@ -31,15 +31,17 @@
     "typescript": "^5.1.6"
   },
   "resolutions": {
+    "@testing-library/jest-dom": "^5.16.5",
```

Comment:
After updating most deps to match with Doodle's newer versions, `react-window` was not resolving to require React v19 and we are still on 18. React Window is only used in bh-shared-ui, not doodle-ui, so we did not see this issue until merging Doodle into the monorepo. These `@types/react[-dom]`resolutions fix the issue. The rest are also alpha sorted.